src/cdomains/apron/octApronDomain.apron.ml

```diff
@@ -525,8 +525,8 @@ struct
     else
       match eval_interval_expr d e with
       | (Some min, Some max) -> ID.of_interval ik (min, max)
```

Comment:
Can overflows not be a problem in the case where octApron knows both bounds? The problem that I've mentioned about `assign_var_handling_underflow_overflow` probably exists here as well: if an intermediate expression overflows but the result fits the type, then this will claim a non-top interval as well.